src/imitation/scripts/train_imitation.py

```diff
@@ -9,12 +9,14 @@
 from sacred.observers import FileStorageObserver
 
 from imitation.algorithms.dagger import SimpleDAggerTrainer
+from imitation.algorithms.sqil import SQIL
 from imitation.data import rollout, types
 from imitation.scripts.config.train_imitation import train_imitation_ex
 from imitation.scripts.ingredients import bc as bc_ingredient
 from imitation.scripts.ingredients import demonstrations, environment, expert
 from imitation.scripts.ingredients import logging as logging_ingredient
 from imitation.scripts.ingredients import policy_evaluation
+from imitation.util.util import save_policy
```

Comment:
Our style guide https://google.github.io/styleguide/pyguide.html#22-imports says:
> Use import statements for packages and modules only, not for individual classes or functions.

(It seems there are some existing violations of this in the file, so easy to have missed this, but still let's not add to them and fix existing issues where possible.)

Reply:
Fixed this and other imports in `train_imitation.py`